src/main/java/com/exasol/errorcodecrawlermavenplugin/config/ErrorCodeConfig.java

```diff
@@ -0,0 +1,77 @@
+package com.exasol.errorcodecrawlermavenplugin.config;
+
+import java.util.*;
+
+import com.exasol.errorreporting.ExaError;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This class represents the contents of the errorCodeConfig.yml. It is created by {@link ErrorCodeConfigReader}.
+ */
+public class ErrorCodeConfig {
+    private final Map<String, SingleErrorCodeConfig> errorTags;
+    private final Map<String, String> packageToErrorCodeMapping;
+
+    @JsonCreator
+    public ErrorCodeConfig(@JsonProperty("error-tags") final Map<String, SingleErrorCodeConfig> errorTags) {
+        this.errorTags = errorTags;
+        this.packageToErrorCodeMapping = inverseMapping(errorTags);
+    }
+
+    private Map<String, String> inverseMapping(final Map<String, SingleErrorCodeConfig> errorTags) {
```

Comment:
can we move this method to the `ErrorCodeConfigReader`, which is the one that actually creates the `ErrorCodeConfig`?